### PR TITLE
Add onboarding tutorial

### PR DESCRIPTION
**Pull request by churichard**

### CODE REVIEW

**Review by danielleung27 — APPROVED** [review]
Looks good!

File: tigerpath/static/app/js/tutorial.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  let intro = introJs();
+  intro.setOptions({
+    showStepNumbers: false,
+    steps: [
+      {
+        intro: "Welcome to TigerPath! We'll take you on a brief tour to show you how to use the app. Click \"Next\" to begin."
+      },
+      {
+        element: '#search-courses',
+        intro: 'You can search for courses you want to add to your schedule here.',
```
Comment (bnehoran) [inline tigerpath/static/app/js/tutorial.js:11]: Maybe add a comment about the Fall/Spring course coloring.

For instance,
`'You can search for courses you want to add to your schedule here. Fall courses appear in [color], and Spring courses appear in [other color].'`

File: tigerpath/static/app/js/tutorial.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        intro: "Search results will appear here. You can drag courses from the search results to any semester in your schedule."
+      },
+      {
+        element: '#semesters',
+        intro: "This is your schedule. It shows all of the courses that you've added to it. You can drag courses between semesters to rearrange your schedule."
+      },
+      {
+        element: '#requirements',
+        intro: `Each course that you've added to your schedule will automatically show up underneath the requirements that it can satisfy.
+        If a course can satisfy multiple requirements, it will show up underneath each of those requirements and be grayed out.
```
Comment (bnehoran) [inline tigerpath/static/app/js/tutorial.js]: If a course can only satisfy one of multiple requirements, ...

File: tigerpath/static/app/js/tutorial.js (hunk 1 above)
Comment (bnehoran) [reply tigerpath/static/app/js/tutorial.js:11]: Or maybe it should go under display-courses? What do you think?

Comment (churichard) [reply tigerpath/static/app/js/tutorial.js:11]: Probably display-courses

**Review by bnehoran — APPROVED** [review]
Awesome!

**Pull request merged**